buildenv/jenkins/testJobTemplate

```diff
@@ -225,9 +225,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJDK_SHA', "", "OpenJDK SHA from which to run tests")
 							stringParam('TKG_OWNER_BRANCH', "AdoptOpenJDK:master", "TestKitGen git [owner]:[branch]. Default is AdoptOpenJDK:master")
 							stringParam('TKG_SHA', "", "TKG sha")
+							stringParam('ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH', "Adoptium:master", "Personal Adoptium/aqa-systemtest repository owner:branch")
```

Comment:
Please use lower case `adoptium` to match with the repo. The same for STF.
https://github.com/adoptium/aqa-systemtest
https://github.com/adoptium/stf